src/network/NetworkManagerSecondary.cc

```diff
@@ -167,6 +167,7 @@ void NetworkManagerSecondary::OnStep(
   msgs::SerializedStateMap stateMsg;
   if (!entities.empty())
     this->dataPtr->ecm->State(stateMsg, entities);
+  stateMsg.set_has_one_time_component_changes(this->dataPtr->ecm->HasOneTimeComponentChanges());
```

Comment:
This line is too long, the codechecker will probably complain

Reply:
Thanks, I thought that `colcon test` was running linters but that doesn't seem to be the case.